### ISSUE TITLE
COPY failed: forbidden path outside the build context: ../.env

### ISSUE THREAD

**Issue by kmeaw** [kirichenko-o/md5-file-hasher#1]
https://github.com/kirichenko-o/md5-file-hasher/blob/794b99c4f12bf7f90e41b26b5b16f7d51066fd48/api.Dockerfile#L8

```
Building api
Sending build context to Docker daemon  120.8kB
Step 1/12 : FROM python:latest
latest: Pulling from library/python
627b765e08d1: Pull complete 
c040670e5e55: Pull complete 
073a180f4992: Pull complete 
bf76209566d0: Pull complete 
ca7044ed766e: Pull complete 
7b16520e0e66: Pull complete 
e121e5a178df: Pull complete 
abbaf10bd160: Pull complete 
87cbe84263a0: Pull complete 
Digest: sha256:2ac6e111d4e5270a0908b3cbfb6a80642233807ea5b098bb2d648478114bb635
Status: Downloaded newer image for python:latest
 ---> cba42c28d9b8
Step 2/12 : WORKDIR /usr/src
 ---> Running in f0125b2b6fe6
Removing intermediate container f0125b2b6fe6
 ---> 190ad7d46853
Step 3/12 : ENV PYTHONDONTWRITEBYTECODE 1
 ---> Running in 405e37e3ed96
Removing intermediate container 405e37e3ed96
 ---> f1bdeb6cde93
Step 4/12 : ENV PYTHONUNBUFFERED 1
 ---> Running in b47018d73481
Removing intermediate container b47018d73481
 ---> 6e966e4084d6
Step 5/12 : COPY /../.env .
COPY failed: forbidden path outside the build context: ../.env ()
ERROR: Service 'api' failed to build : Build failed
```

**Comment by kmeaw** [comment]
```
% docker version
Client:
 Version:           20.10.7
 API version:       1.41
 Go version:        go1.16.5
 Git commit:        f0df35096d5f5e6b559b42c7fde6c65a2909f7c5
 Built:             Sat Jun  5 23:38:09 2021
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server:
 Engine:
  Version:          20.10.7
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.16.5
  Git commit:       b0f5bc36fea9dfb9672e1e9b1278ebab797b9ee0
  Built:            Sat Jun  5 23:20:28 2021
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          v1.5.4
  GitCommit:        a62e1d690afa2b9b1d43f8ece3ff4483
 runc:
  Version:          1.0.1
  GitCommit:        4144b63817ebcc5b358fc2c8ef95f7cddd709aa7
 docker-init:
  Version:          0.19.0
  GitCommit:        
```

```
% docker-compose --version
docker-compose version 1.29.2, build unknown
```

**Comment by kmeaw** [comment]
This issue seems to be fixed in eb93ef68a21f8f63d9960ab7f229ed874f7a8854.
